lib/python/scalgoproto.py

```diff
@@ -438,16 +470,24 @@ def __init__(self, offset: int) -> None:
 class TableOut(object):
     __slots__ = ["_writer", "_offset"]
     _MAGIC: ClassVar[int] = 0
+    _SIZE: ClassVar[int] = 0
+    _DEFAULT: ClassVar[bytes] = b""
 
-    def __init__(self, writer: "Writer", with_weader: bool, default: bytes) -> None:
+    def __init__(
+        self, writer: "Writer", with_header: bool = True, offset: Optional[int] = None
```

Comment:
This breaks the current dependant code. 

In our own `run.py`, we have the following.
```
super().__init__(writer, withHeader,  b"\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0")
```